xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+    <screen>(each controller)> ps -ef | egrep 'cinder-volume|nova-consoleauth'</screen>
+   </listitem>
+  </itemizedlist>
+</step>
+<step>
+  <para>
+   Prepare the cloud for migration:
+  </para>
+  <itemizedlist>
+  <listitem>
+   <para>
+    Ensure that the data model enables deployer behavior by the new seed
+    node.
+   <remark>taroth 2018_02-27: what is this supposed to mean?</remark>
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      In the <filename>disks_*.yml</filename> file used by your server role
+      that includes monitoring services, replace the existing <literal>Vertica</literal>
+      partition definition with two <literal>Cassandra</literal> partitions:
+     </para>
+   <screen>- name: cassandra
+            size: (whatever size was used originally by vertica partition, less 1%)
+            mount: /var/cassandra/data
+            fstype: ext4
+            mkfs-opts: -O large_file
+            consumer:
+              name: vertica
+          - name: cassandra_log
+            size: 1%
+            mount: /var/cassandra/commitlog
+            fstype: ext4
+            mkfs-opts: -O large_file
+   </screen>
+    </listitem>
+    <listitem>
+     <para>
+      Edit
+      <filename>~/helion/my_cloud/definition/data/control_plane.yml</filename>
+      and identify the cluster definition that already includes an entry for
+      <filename>mysql</filename> in its service-components.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Ensure that <systemitem>lifecycle-manager</systemitem> is included in the same
+      cluster's service-components list. If it was necessary to add this as a
+      new entry, it will also need to be removed from its original position in
+      the deployer.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Add a new entry for <systemitem>nova-placement-api</systemitem> alongside whichever cluster
+      already includes <systemitem>nova-api</systemitem>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Commit your changes to the git model:
+     </para>
+     <screen>(deployer)> git commit -a -m "Prepare for HOS 5.1 installation"</screen>
+    </listitem>
+   </itemizedlist>
+  </listitem>
+  <listitem>
+   <para>
+    Consider the effect of control plane downtime on existing DHCP leases:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Default DHCP lease time is 48 hours as defined in the
+      <filename>~/helion/hos/ansible/roles/neutron-common/templates/neutron.conf.j2</filename> file by the
+      <systemitem>dhcp_lease_duration</systemitem> parameter.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      DHCP leases are administered by &o_netw; and dnsmasq. This functionality will not be available
+      during the control plane downtime.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      DHCP clients typically renew their leases either 50% of the way into the lease time or upon
```

Comment:
simplification: "DHCP clients typically renew their leases either after 50% of the lease time or upon reboot."